.github/workflows/scripts/slack_gem_notifications/notifications_methods.rb

```diff
@@ -17,7 +17,7 @@ def check_for_updates(watched_gems)
 
```

Comment:
This just occurred to me: Do you have a README for `slack_gem_notifications`? I couldn't find one with a quick look, however, it might be helpful for posterity!

Reply:
I do not! I can add one. Good idea

Reply:
resolved with [aea8bf2](https://github.com/newrelic/newrelic-ruby-agent/pull/1273/commits/aea8bf24a85127371cc0917c9af427bca18e1573)